src/database/migrations/0032-insert-system-source.sql

```diff
@@ -0,0 +1,11 @@
+INSERT INTO sources ( label, source_type ) VALUES ( 'PDC', 'system' );
```

Comment:
We could -- though I do personally think that a function is generally less brittle.

It's possible someone could run the migrations up to right before this one, insert a row, and then this migration would fail AND the `1` would become incorrect. Or they could blow away the data and re-insert the system user and it would have a non-1-id.

Basically, I think the function is safer than a hard coded reference to something that is dynamically populated.

Reply:
Since you already did the work, :+1: